docker-compose.example.yml

```diff
@@ -14,9 +15,12 @@ services:
       - DJANGO_SETTINGS_MODULE=babybuddy.settings.docker
       - SECRET_KEY=
       - TIME_ZONE=
+      - POSTGRES_PASSWORD=postgres
+      - DEBUG=False # Turn to False in production
     volumes:
       - media:/app/media:rw
-    command: gunicorn -c /app/gunicorn.py babybuddy.wsgi
+    # Sleep 5 seconds to allow the db to to come up;
+    command: bash -c 'sleep 5 && python manage.py migrate --noinput && python manage.py createcachetable  && python manage.py collectstatic --noinput && gunicorn babybuddy.wsgi -b :8000 --log-level=info'
```

Comment:
One small change here, please remove the "collectstatic" portion of this command. Baby Buddy's static assets are bundled with the repository.